### ISSUE TITLE
formnovalidate is not working

### ISSUE THREAD

**Issue by rajeshkumarp89** [formvalidation/support#174]
Hi there,

I am integrating formvalidation in one of the ASP.Net application, I have given formnovalidate tag for the button which does not require validation and it is not working in IE. In IE when I click the button which has formnovalidate page just reloads without any validation message or the desired action. Hope it helps.

![image](https://cloud.githubusercontent.com/assets/5183291/8104800/9443d5e4-102b-11e5-991f-cce6509235f7.png)

Thanks,
Rajesh

**Comment by nghuuphuoc** [comment]
By default, clicking on the button with `formnovalidate` attribute will skip the validation process.
Take a look at the [Skip validation on specific submit button](http://formvalidation.io/examples/skip-validation-specific-submit-button/) example.

**Comment by rajeshkumarp89** [comment]
Yes, it is skipping the validation but the submit is not happening. I have found a similar issue https://github.com/formvalidation/support/issues/131 in the issue list. I am experiencing the same kind.

Thanks,
Rajesh

**Comment by nico66** [comment]
I join in the request, yet I have the exact same problem on the internet explor.

**Comment by nico66** [comment]
There is some news for this bug?
Unfortunately I'm stuck.

thanks

**Comment by rajeshkumarp89** [comment]
I resolved my issue by adding UseSubmitBehavior="false" for the asp Button

**Comment by nico66** [comment]
I've tried works.
It can work as a temporary workaround!

https://msdn.microsoft.com/it-it/library/system.web.ui.webcontrols.button.usesubmitbehavior%28v=vs.110%29.aspx

**Comment by dferreira1** [comment]
I'm stucked on it as well...

**Comment by nico66** [comment]
Some news?

**Comment by nico66** [comment]
Help!!! :-)

**Comment by nico66** [comment]
I'm starting another project .. does not have any good news?

**Comment by nghuuphuoc** [comment]
Not yet. I haven't found the solution for it except using the `UseSubmitBehavior` attribute as you mentioned above.

**Comment by nico66** [comment]
Thank you.
Good job.

**Comment by dtkujawski** [comment]
I have experienced the same behavior (not using ASP though).  The formnovalidate attribute does not work in IE11 (same screen works fine in Chrome and Firefox)

**Comment by dtkujawski** [comment]
I found a solution to this by changing the _init method in base.js from:
```html
                        // Don't perform validation when clicking on the submit button/input which
                        // aren't defined by the 'button.selector' option
                        if (that.options.button.selector && !$button.is(that.options.button.selector) && !$button.is(that.$hiddenButton)) {
                            that.$form.off('submit.' + that._namespace).submit();
                        }
```

to the following:
```html
                        // Don't perform validation when clicking on the submit button/input which
                        // aren't defined by the 'button.selector' option
                        if (that.options.button.selector && !$button.is(that.options.button.selector) && !$button.is(that.$hiddenButton)) {
                            that.$form.append($('<input/>').attr('type', 'hidden').attr('name', $button.attr('name')).val($button.val()));
                            that.$form.off('submit.' + that._namespace).submit();
                            return false;
                        }
```

**Comment by srianbury** [comment]
Any new discoveries?
